### ISSUE TITLE
Request: Data Processing Enhancements

### ISSUE THREAD

**Issue by AjeetAthwal** [pyrochlore/obsidian-tracker#303]
I would like to create a moving average on some metrics that I track daily to smooth out the trends. Looking at the roadmap, I can see there's a section called "Data Processing" with a bullet point on post-process functions. I'd just like to prioritise it.

**Comment by lazyguru** [comment]
I don't think that roadmap is relevant any more. It hasn't been followed in almost 2 years at this point. Features are contributed by whomever is willing to work on them. I have merge access and will review+merge any feature contributions. I will pickup a bug fix here or there, but I don't have any current plans to contribute any features myself.

**Comment by AjeetAthwal** [comment]
Thanks for the response.

I'll see if I have time to look at it and see if I can contribute
